### PR TITLE
Json - Minor changes in preparation for io hash JSON export.

### PR DESCRIPTION
**Pull request by rd2**

### CODE REVIEW

File: lib/measures/tbd/measure.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def run(model, runner, user_arguments)
       return false
     end
 
-    surfaces = processTBD(model, set)
+    io, surfaces = processTBD(model, set)
     surfaces.each do |id, surface|
```
Comment (rd2) [inline lib/measures/tbd/measure.rb:63]: For JSON export, the easiest and safest solution is to prune/edit the _io_ hash once the derated _surfaces_ objects have their _:ratio_ values calculated (and therefore right before both objects are returned at the end of processTBD). Initial tests seem to work just fine.

File: lib/measures/tbd/resources/psi.rb [hunk 2]
```diff
@@ -161,7 +161,7 @@ def complete?(s)
   end
 end
 
-def processTBDinputs(surfaces, edges, set, io_path, schema_path)
+def processTBDinputs(surfaces, edges, set, io_path = nil, schema_path = nil)
```
Comment (rd2) [inline lib/measures/tbd/resources/psi.rb:164]: As per previous PR request.

File: spec/tbd_tests_spec.rb [hunk 3]
```diff
@@ -1979,6 +1979,12 @@
       next unless id == "Entryway  Wall 5"
       expect(surface[:heatloss]).to be_within(0.01).of(13.96)
     end
+
+    #out = JSON.pretty_generate(io)
+    #out_path = File.dirname(__FILE__) + "/../json/tbd_minimal_KHI.out.json"
+    #File.open(out_path, "w") do |out_path|
+    #  out_path.puts out
+    #end
```
Comment (rd2) [inline spec/tbd_tests_spec.rb:1987]: The final version of the _io_ object may be tweaked a few times over the next 48 hours. For now, let's stick its current state.

File: lib/measures/tbd/resources/psi.rb (hunk 2 above)
Comment (macumber) [reply lib/measures/tbd/resources/psi.rb:164]: Thanks!

**Pull request merged**